File: SDKs/shutterstock.com/1.1.5/python/src/sdk/models/shared/licenseeditorialcontentrequest.py
import dataclasses
from typing import Any
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import licenseeditorialcontent as shared_licenseeditorialcontent


@dataclass_json
@dataclasses.dataclass
class LicenseEditorialContentRequest:
    r"""LicenseEditorialContentRequest
    License editorial content request
    """
    
    country: Any = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('country') }})
    editorial: list[shared_licenseeditorialcontent.LicenseEditorialContent] = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('editorial') }})
    

File: SDKs/shutterstock.com/1.1.5/typescript/src/sdk/models/shared/licenseeditorialcontentrequest.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { LicenseEditorialContent } from "./licenseeditorialcontent";



// LicenseEditorialContentRequest
/** 
 * License editorial content request
**/
export class LicenseEditorialContentRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=country" })
  country: any;

  @SpeakeasyMetadata({ data: "json, name=editorial", elemType: LicenseEditorialContent })
  editorial: LicenseEditorialContent[];
}
